docs/articles/documentation/using-testcafe/programming-interface/TestCafe.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: TestCafe Class
+permalink: /documentation/using-testcafe/programming-interface/TestCafe/
+---
+# TestCafe Class
+
+A TestCafe server instance.
+
+To create it, use the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/global/#createTestCafe) global function.
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+```
```

Comment:
We need to describe args, I guess. What do they mean? Moreover, they are all optional.